core/pom.xml

```diff
@@ -370,5 +370,28 @@
             </activation>
             <!-- not including license-maven-plugin is sufficent to expose default license -->
         </profile>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>attach-javadocs</id>
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                                <configuration>
+                                    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
The errors have already been fixed on master (see #13702). There was a big cleanup, it was not just the few fixes in #14766.  You can try cherry picking them; there were 10 or so files that needed manual merging when I glanced at it. But I still would not want to push this change without doclint: we need to ensure we are building good javadocs.